apollo-runtime/src/appleMain/kotlin/com/apollographql/apollo3/network/ws/NSURLSessionWebSocketEngine.kt

```diff
@@ -113,7 +110,7 @@ private class WebSocketConnectionImpl(
   init {
     messageChannel.invokeOnClose {
       webSocket.cancelWithCloseCode(
-          closeCode = CLOSE_NORMAL.toLong(),
+          closeCode = CLOSE_NORMAL.convert(),
```

Comment:
👍

Reply:
Actually I'm curious... Is there a platform where `closeCode` isn't a Long?

Reply:
Yes I think it was `watchosArm64` but let me check to be sure.

Reply:
Yes that's the one:

```
> Task :apollo-runtime:compileKotlinWatchosArm64 FAILED
e: /Users/bod/gitrepo/1/apollo-android/apollo-runtime/src/appleMain/kotlin/com/apollographql/apollo3/network/ws/NSURLSessionWebSocketEngine.kt: (113, 23): Type mismatch: inferred type is Long but NSURLSessionWebSocketCloseCode /* = Int */ was expected
```